_releases/2022-01-06-1.3.0-released.md

```diff
@@ -22,6 +22,21 @@ In this release we included [XXX PRs since the 1.2.2 release](https://github.com
 
 ## Multi-Assign
 
+Multiple assignments got improved in various ways. First, it's possible to take a "splat of elements" in a multi-assignment:
+
+<div class="code_section">{% highlight crystal %}
+n_first, *n_s, n_last = [1, 2, 3, 4, 5]       # n_s is [2, 3, 4]
+*t_s, t_last = {"This", 15, 4, "tuple", true} # t_s is {"This", 15, 4, "tuple"}
+{% endhighlight %}</div>
+
+Second, it's possible to detect unbalanced multi-assignments:
```

Comment:
I'd mention the opt-in restriction right at the start, to make it clear that the following example does not show the default behaviour:
```suggestion
Second, there's an optional preview feature to detect unbalanced multi-assignments. It can be enabled with the compiler flag `-Dstrict_multi_assign`.
```